release/airflow/airflow_scripts.sh

```diff
@@ -97,23 +101,23 @@ function build_template() {
     exit "${BUILD_FAILED}"
 }
 
+# this function is called directly by airflow
 function test_command() {
-    cp /home/airflow/gcs/data/githubctl ./githubctl
-    chmod u+x ./githubctl
-    pwd; ls -l    ./githubctl
+    local KEY_FILE_PATH
+    local WAIT_FOR_RESULT
+    KEY_FILE_PATH=""
```

Comment:
why is this always empty?

Reply:
this is there if we want to override the path in development. We dont use this when the pipeline is actually running

Reply:
Why is it passed to the runbuild comand below if it is not used in development?